CHANGELOG.md

```diff
@@ -7,6 +7,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [0.8.0] - 2023-XX-XX
+
+### Breaking changes
+
+-   Added support for FDI 1.18
+-   User type has changed across recipes and functions: recipe specific user types have been removed and replaced by a generic one that contains more information
+-   `createdNewUser` has been renamed to `createdNewRecipeUser`
+-   `createCode`, `consumeCode`, `createPasswordlessCode` and `consumePasswordlessCode` can now return status: `SIGN_IN_UP_NOT_ALLOWED`
+-   `signInAndUp` and `thirdPartySignInAndUp` can now return new status: `SIGN_IN_UP_NOT_ALLOWED`
+-   `sendPasswordResetEmail` can now return `status: "PASSWORD_RESET_NOT_ALLOWED"`
+-   `signIn` and `emailPasswordSignIn` can now return `SIGN_IN_NOT_ALLOWED`
+-   `signUp` and `emailPasswordSignUp` can now return `SIGN_UP_NOT_ALLOWED`
+
+### Migration
+
+#### New User structure
+
+We've added a generic `User` type instead of the old recipe specific ones. The mapping of old props to new in case you are not using account-linking:
+
+-   `user.id` stays `user.id`
+-   `user.email` becomes `user.emails[0]`
+-   `user.phoneNumber` becomes `user.phoneNumbers[0]`
+-   `user.thirdParty` becomes `user.thirdParty[0]`
+-   `user.timeJoined` is still `user.timeJoined`
+-   `user.tenantIds` is still `user.tenantIds`
+
```

Comment:
Some of the existing APIs return new status -> like sign in returns SIGN_IN_NOT_ALLOWED. Mention all of them.

Reply:
They are mentioned above, I just skipped them here since there is no real migration needed. They cannot be returned when upgrading, they only happen if you also turn on account-linking, which is a separate documentation.